core/templates/components/common-layout-directives/common-elements/background-banner.component.html

```diff
@@ -1,12 +1,13 @@
-<div class="oppia-background-banner position-absolute w-100 text-center">
+<div class="w-100 text-center background-banner-position">
```

Comment:
Why not call this oppia-background-banner? It is a better description -- this should be a noun that describes the thing, not a "position".

Reply:
Done.